src/index.js

```diff
@@ -20,20 +20,34 @@ import Booking from './Booking';
 let allCustomers;
 let allBookings;
 let allRooms;
+let dateSelected;
 
 let userGreeting = document.querySelector('.user-greeting');
 let totalSpent = document.querySelector('.total-spent');
-let bookingsList = document.querySelector('.bookings');
+let bookingsList = document.querySelector('.bookings-list');
+let bookNewRoomButton = document.getElementById('book-new-room');
+let datePicker = document.getElementById('date');
+let datePickerLabel = document.querySelector('.date-picker');
+let dropDown = document.querySelector('.dropdown');
 
 
+function hide(element) {
+  element.classList.add('hidden');
+}
 
+function show(element) {
+  element.classList.remove('hidden');
+}
 
+function reset(element) {
+  element.innerHTML = '';
+}
 
 function fetchData() {
   Promise.all([allCustomerDataAPI, bookingDataAPI, roomDataAPI])
     .then((values) => {
       createInstances(values);
-      renderUserDashboard()
+      renderUserDashboard();
     })
 }
 
```

Comment:
Really interested in how you got this to work considering I've been unsuccessful with my global hookups